google/cloud/aiplatform/experimental/vertex_model/base.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import abc
+from concurrent import futures
+import datetime
+import functools
+import inspect
+import logging
+import pandas as pd
```

Comment:
Standard lib and 3rd party library imports should be in separate blocks.

Also, since `pandas` and `pytorch` are not required by the package, they should be imported with exception blocks.

```python
try:
  import pandas as pd
except ImportError:
 raise ImportError("Pandas is not installed. Please install pandas to use VertexModel")
```

We can follow up and add an extras_requires in the future.